packages/ethereum-storage/test/ethereum-storage.test.ts

```diff
@@ -528,14 +528,6 @@ describe('EthereumStorage', () => {
       await expect(ethereumStorage.read('')).rejects.toThrowError('No id provided');
     });
 
-    it('append and read on an invalid ipfs gateway should throw an error', async () => {
```

Comment:
I don't understand why we have such a method, I removed it, it's only used in this test